skeleton-typescript-webpack/tsconfig.json

```diff
@@ -9,6 +9,8 @@
   "filesGlob": [
     "./src/**/*.ts",
     "./node_modules/aurelia-*/dist/aurelia-*.d.ts",
+    "./custom_typings/**/*.d.ts",
```

Comment:
Yes, that potentially sounds better - I'm not sure what the convention for this would normally be. I had the folder in that location primarily to mirror the structure of the [typescript-jspm skeleton](https://github.com/aurelia/skeleton-navigation/tree/master/skeleton-typescript).